tests/client/components/__snapshots__/carousel.spec.js.snap

```diff
@@ -258,7 +258,7 @@ exports[`Carousel should reset autoplay when switching slide without autoplayHov
 
 exports[`Carousel should set carousel height to slide height 1`] = `
 ".VueCarousel-wrapper
-  .VueCarousel-inner(role='listbox', style='transform: translate(0px, 0); visibility: hidden; height: 200px;')
+  .VueCarousel-inner(role='listbox', style='transform: translate(0px, 0); visibility: hidden; height: auto;')
```

Comment:
With vue-test-utils, the html doesn't seems to get updated even after `$nextTick`.
If anyone knows what's up with this, your input is very much welcomed.

Reply:
@ashleysimpson any idea about this? I've encountered similar stuff in our internal tests but we're working around it in the same manner. Not sure if there is a better alternative

Reply:
@josephting after a little bit of research, if you add the property `sync: false` when you mount the wrapper the html will update as expected

Reply:
@ashleysimpson That was a huge help. Thank you so much!
@quinnlangille With that, this PR will only have changes to the test scripts and the snapshots will be untouched.